### PR TITLE
Add configuration for materialized view consistency

### PR DESCRIPTION
**Pull request by zacw7**
Changes include:
1. Create a configuration `materialized_view_data_consistency_enabled` to control partition stitching. This config is on by default.
2. Update the corresponding unit test.

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java [hunk 1]
```diff
@@ -197,6 +197,7 @@
     public static final String PARTIAL_RESULTS_MAX_EXECUTION_TIME_MULTIPLIER = "partial_results_max_execution_time_multiplier";
     public static final String OFFSET_CLAUSE_ENABLED = "offset_clause_enabled";
     public static final String VERBOSE_EXCEEDED_MEMORY_LIMIT_ERRORS_ENABLED = "verbose_exceeded_memory_limit_errors_enabled";
+    public static final String PARTITION_STITCHING_ENABLED = "partition_stitching_enabled";
```
Comment (jainxrohit) [inline presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java:200]: Lets call it MATERIALIZED_VIEW_DATA_CONSISTENCY_ENABLED

**Review by jainxrohit — CHANGES_REQUESTED** [review]
Looks good.

We follow following commit guidelines when writing commit message
https://chris.beams.io/posts/git-commit/

Lets change the commit message and description as 

Add configuration for materialized view consistency

Adding a configuration to control the materialized view data consistency. When enabled and reading from materialized view, partition stitching is applied to achieve data consistency.

File: presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java (hunk 1 above)
Comment (zacw7) [reply presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java]: Renamed the config accordingly.

**Review by highker — APPROVED** [review]

**Comment by ajaygeorge** [comment]
@zacw7 This PR is missing Release notes.
Can you please add the Release notes as described by the guidelines [here](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) .
cc @jainxrohit @highker

**Comment by zacw7** [comment]
> @zacw7 This PR is missing Release notes.
> Can you please add the Release notes as described by the guidelines [here](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) .
> cc @jainxrohit @highker

Updated. Thanks Ajay!

**Pull request merged**